cmd/server/customer_search.go

```diff
@@ -55,6 +56,11 @@ func readSearchParams(u *url.URL) searchParams {
 	if limit, err := strconv.ParseInt(util.Or(u.Query().Get("limit"), "20"), 10, 32); err == nil {
 		params.Limit = limit
```

Comment:
Maybe we can limit the limit? Let's say not more than 100 records? Without limiting the number of results someone can attack us (DoS).

Reply:
Yes. Jordan is working on a [helper method for all projects](https://github.com/moov-io/base/pull/79).